.github/workflows/ci.yaml

```
@@ -222,7 +222,9 @@ jobs:
                   coverage: none
             
             - name: Add solr dependency
-              run: composer require --no-update "ibexa/solr:~4.4.0@dev"
+              run: |
+                VERSION=$(cat composer.json | jq -r '.extra | ."branch-alias" | ."dev-main"')
```

You don't have to involve `cat` :cat:  in this process, `jq` can read standard input
```suggestion
                VERSION=$(jq -r '.extra | ."branch-alias" | ."dev-main"' < composer.json)
```

---

Thanks!